src/Bicep.Core/Semantics/Namespaces/SystemNamespaceSymbol.cs

```diff
@@ -1,12 +1,12 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
-using System;

```

Comment:
The whitespace diffs are due to line ending normalization. The file has mixed line endings and I changed it to LF.